src/components/App/App.tsx

```diff
@@ -344,6 +344,9 @@ export default class App extends React.Component<AppProps, AppState> {
       this.updateName(null, { value: userName || generateName() });
       this.loadSignInData();
     });
+    socket.on('kicked', () => {
```

Comment:
I feel like we should just silently send the user to the home page. No need to explicitly mention that they've been kicked.